test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSTextField.cs

```diff
@@ -1903,5 +1903,50 @@ public void TextFieldCharacterSpacing()
             testingTarget.Dispose();
             tlog.Debug(tag, $"TextFieldCharacterSpacing END (OK)");
         }
+
+        [Test]
+        [Category("P1")]
+        [Description("TextField PrimaryCursorPosition.")]
+        [Property("SPEC", "Tizen.NUI.TextField.PrimaryCursorPosition A")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "CONSTR")]
+        [Property("AUTHOR", "s.sabah@samsung.com")]
+        public void TextFieldPrimaryCursorPosition()
```

Comment:
Hello @shrouqsabah 
You know, `TextFieldPrimaryCursorPosition()` test already exists in current test code.
But I think your test is better because it can cover not only positive cases but also out of range cases.

So, in the rebase, please replace the existing testcase with your test.

```
// current TextFieldPrimaryCursorPosition test
        [Test]
        [Category("P1")]
        [Description("TextField PrimaryCursorPosition.")]
        [Property("SPEC", "Tizen.NUI.TextField.PrimaryCursorPosition A")]
        [Property("SPEC_URL", "-")]
        [Property("CRITERIA", "CONSTR")]
        [Property("AUTHOR", "bowon.ryu@samsung.com")]
        public void TextFieldPrimaryCursorPosition()
```

Reply:
@wonrst , Done, Could you check it , please.
Thanks :)